packages/gitmoji-changelog-core/src/index.js

```
@@ -84,7 +92,14 @@ function sortVersions(c1, c2) {
 
 function hasNextVersion(tags, release) {
   if (!release || release === 'next') return true
-  return tags.some(tag => semver.eq(tag, release))
+  try {
+    return tags.some(tag => semver.eq(tag, release))
+  } catch (e) {
+    if (e.name === 'TypeError') {
+      throw FunctionalError(e.message)
+    }
+    return null
```

You may should throw the error here instead of returning null ?
`throw e`